architecture/adr-list/adr-008-identity-resources.md

```diff
@@ -67,40 +81,56 @@ controller(s). All DIDs must exist.
   "attrNames": ["undergrad", "last_name", "first_name", "birth_date", "postgrad", "expiry_date"]
 }
 ```
+This creates an artifact, that when dereferenced will return the appropriate fields for the schema. 
 
-Don't store Schema DIDDoc in the State.
+#### Schema DID Document URL
 
-Schema URL: `did:cheqd:mainnet-1:N22KY2Dyvmuu2PyyqSFKue`
-
-Schema Entity URL: `did:cheqd:mainnet-1:N22KY2Dyvmuu2PyyqSFKue?service=CL-Schema`
```

Comment:
Fully agree that we need to update a schema URI definition. But I suggest not to remove it.